README.md

```diff
@@ -1 +1,45 @@
-# planetary-pub2
\ No newline at end of file
+# planetary-pub2
+
+GraphQL types and resolvers for a planetary public server
+
+## Usage
+
+`npm install -S planetary-pub2`
+
+## API
+
+### Queries
+
+```
+# Message Fragment
+{
+  key
+    value {
+      previous
+      sequence
+      author
+      timestamp
+      hash
+      content {
+        type
+        text
+        root
+      }
+      signature
+    }
+  }
+}
+```
+
+```
+getPost(key: ID) {
+  message {
+    ...MessageFragment
+  }
+  threads {
+    ...MessageFragment
+  }
+  full
+}
```

Comment:
out of date. Just delete it all I reckon.
The server starts a playground which is basically living docs